app/workers/delete_plain_object_worker.rb

```diff
@@ -9,8 +9,11 @@ class DeletePlainObjectWorker < ActiveJob::Base
   end
 
   rescue_from(ActiveRecord::RecordNotDestroyed, ActiveRecord::StaleObjectError) do |exception|
-    System::ErrorReporting.report_error(exception, parameters: { caller_worker_hierarchy: caller_worker_hierarchy,
-                                                                 error_messages: exception.record.errors.full_messages })
+    record = exception.record
+    if record.class.exists?(record.id)
```

Comment:
It would be nice to have a test about this.
It is a race condition I think

Reply:
I tried to add a test and I couldn't reproduce it as I said in the description.

Reply:
yeah, test would be nice.

Reply:
@hallelujah @macejmic I agree, but how? 😝 It is not that easy 😞